src/logius/inlines.js

```diff
@@ -0,0 +1,349 @@
+// @ts-check
+// Module core/inlines
```

Comment:
Likewise, we might be able to add the bits you need to the core one? This is using things you probably don't need (e.g., the WebIDL inline parser)

Reply:
moved nl stuff to core version.